src/surfaces/filtersDrawer/filtersDrawerNestedTogglesValueLabel.jsx

```diff
@@ -1,10 +1,13 @@
+import { get, noop, size } from 'lodash';
```

Comment:
Up to @cbrewer1689 as the main steward of this codebase as to whether this is Blocker or NAB (and maybe @War777 can weigh in as well) but I think we tend to prefer one thing per line these days:
```JavaScript
import {
    get,
    noop,
    size,
} from 'lodash';
```

Reply:
Fixed.

Reply:
Thank you good sir!  Resolving.